templates/ts/client.ts

```diff
@@ -91,7 +94,13 @@ export class {{name}}Client {
           {{/clientContext}}
         }
       });
-      return coerceWithSchema(this.props.{{{name}}}.properties.returns, ret, schema) as {{{returnType}}};
+
+      const validator = this.validators.{{{name}}};
```

Comment:
This upgrades all `-client`s to perform both validation and coercion. This probably shouldn't break anything, but we should make sure to upgrade them to verify they are working.